data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -54,9 +48,167 @@
+                        count=1
+                    [/have_unit]
+                    [then]
+                        [store_unit]
+                            [filter]
+                                x,y=18,18
+                            [/filter]
+                            variable=twin
+                        [/store_unit]
+                        [switch]
+                            variable=side_number
+                            [case]
+                                value=1
+                                {VARIABLE x 3}{VARIABLE y 34}
+                            [/case]
+                            [case]
+                                value=2
+                                {VARIABLE x 33}{VARIABLE y 35}
+                            [/case]
+                            [case]
+                                value=3
+                                {VARIABLE x 33}{VARIABLE y 3}
+                            [/case]
+                            [case]
+                                value=4
+                                {VARIABLE x 3}{VARIABLE y 3}
+                            [/case]
+                        [/switch]
+                        [lift_fog]
+                            [filter_side]
+                                side=$side_number
+                            [/filter_side]
+                            x,y=$x,$y
+                            radius=2
+                        [/lift_fog]
+                        # A short delay to allow the bonus income animation time to play before scrolling away.
+                        [delay]
+                            time=1000
+                            accelerate=yes
+                        [/delay]
+                        [scroll_to]
+                            x,y=$x,$y
+                        [/scroll_to]
+                        [unit]
+                            side=$twin.side
+                            type=$twin.type
+                            canrecruit=yes
+                            x,y=$x,$y
+                            placement=map
+                            animate=yes
+                        [/unit]
+                        [floating_text]
+                            x,y=$x,$y
+                            text= _ "<span color='white'>Heir Apparent</span>"
+                        [/floating_text]
+                        [delay]
+                            time=1500
+                            accelerate=yes
+                        [/delay]
+                    [/then]
+                [/if]
+            [/then]
+        [/if]
+    [/event]
+
+    # A leader unlocks the hill treasure chest and takes its gold.
+    {PLACE_IMAGE items/chest.png 18 18}
+    [event]
+        name=moveto
+        [filter]
+            x,y=18,18
+            canrecruit=yes
+        [/filter]
+        [gold]
+            amount=$chest_gold
+            side=$unit.side
+        [/gold]
+        {REMOVE_IMAGE $x1 $y1}
```

Comment:
This looks like it would remove all images on this tile, which could break add-ons/modifications that want to place images

Reply:
See `[remove_item]` for how to avoid that.

Reply:
Good suggestions. Applied.